src/Illuminate/Encryption/Encrypter.php

```diff
@@ -52,7 +56,7 @@ public function encrypt($value)
     {
         $iv = openssl_random_pseudo_bytes($this->getIvSize());
 
-        $value = base64_encode($this->padAndEncrypt($value, $iv));
+        $value = openssl_encrypt(serialize($value), $this->cipher, $this->key, 0, $iv);
```

Comment:
I think the docblock should be changed into `* @param  mixed  $value`.
What do you think @GrahamCampbell ?